### DISCUSSION TITLE
Delivery customization functions doesn't work stable.

### DISCUSSION THREAD

**Discussion by letmepro2610** [Shopify/function-examples#385] [Feedback: Delivery Customization API]
I'm getting an issue with my app on my development store. The first time I installed the app in my development store, the function worked properly. After that, I removed the app and reinstalled it later, the Shopify function doesn't work anymore.  I have checked the Admin graphql query **shopifyFunctions**, the functions are still here but it does not show any login in App Insight. 
Note: If I install it on a new site, the function will work normally. 

Is there anyone pls tell me how can I fix this bug?

**Comment by nickwesselman** [comment]
If you uninstall the app, then your Payment Customization under Settings > Payments > Payment customizations would have been deleted. After you reinstalled, did you use a UI or GraphQL mutation to recreate the payment customization in your development store?

**Reply by letmepro2610** [reply]
hi @nickwesselman ,
Thank you for your comment. I have found the issue is I have not declared Shopify functions again when reinstalling the app.

**Comment by n8cotoa** [comment]
@nickwesselman Are there future plans to have enabling of Delivery and Payment Customizations functions/rules within the native Shopify Admin UI? (similar to Cart/Checkout Validations)

**Reply by nickwesselman** [reply]
I'd suggest posting a new feature request for this, describing the use case.
